class AcknowledgeError(Exception):
    pass

class AlreadyExistsError(Exception):
    pass

class NotFoundError(Exception):
    pass
